patientsearch/src/js/components/UrineScreen.js

```diff
@@ -294,7 +294,12 @@ export default function UrineScreen(props) {
                 "reference": "Patient/"+(params.patientId ? params.patientId : rowData.id)
             }
         };
-    };
+        //include id if present, necessary for PUT request
+        if (params.id) {
+            resource = {...resource, ...{"id": params.id}};
+        }
+        return resource;
+    }
```

Comment:
I think the CI is nagging about a semicolon here?
```suggestion
    };
```

Reply:
thank you @ivan-c  This should be fixed.